src/DiagnosticsModule.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace Carter
+{
+    using System;
+    using System.Linq;
+    using Carter.Response;
+
+    internal class DiagnosticsModule : CarterModule
+    {
+        public DiagnosticsModule(CarterDiagnostics diagnostics) : base("/_CarterDiagnostics")
+        {
+            this.Get("/", async context =>
+            {
+                Console.WriteLine("woo");
+                var responseObject = new

```

Comment:
This is the bit I dislike the most - would love some input on what this should look like / what else it could include